src/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/ValueToItems.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.Android.Tools.BootstrapTasks
+{
+	public class ValueToItems : Task
+	{
+		[Required]
+		public  string              Value                   { get; set; }
+
+		[Required]
+		public  string              Separator               { get; set; }
+
+		public  ITaskItem[]         Exclude                 { get; set; }
```

Comment:
You can use a `string[]` type here, since you don't need the metadata from the items.